projects/canopy/src/lib/forms/radio/radio-group.component.ts

```diff
@@ -117,7 +119,11 @@ export class LgRadioGroupComponent implements ControlValueAccessor {
     @SkipSelf()
     private controlContainer: FormGroupDirective,
     private domService: LgDomService,
+    private hostElement: ElementRef,
   ) {
+    this.variant = this.hostElement.nativeElement.tagName
```

Comment:
Just wondering if we still need variant as an `@Input()` as it's automatically set here? Also, if the user sets a variant via the input, this might overwrite it. Do we want the user to manually set it? And if so, perhaps their input should take precedence.

Reply:
the `@Input()` was left behind from before, when I tried the implementation where the user would set it manually. I will remove it as the current implementation should give the user the same possibilities by using `lg-radio-group` or `lg-filter-group`. I think if we were to let both implementations that would be misleading to the user, and would actually try to use both, just a thought.

Reply:
Yeah I think it could confusing for the user. What do you think @thatguynamedandy  - shall we drop the `@Input` entirely? Or figure out how to allow the user to manually set the variant via the `@Input` as well?

Reply:
variant is not a `@Input()` property anymore. :)